raiden/transfer/views.py

```diff
@@ -311,6 +295,21 @@ def get_channelstate_by_token_network_and_partner(
     return channel_state
 
 
+def get_current_claims_by_token_network_and_partner(
+    chain_state: ChainState, token_network_address: TokenNetworkAddress, partner: Address
+) -> Tuple[Claim, Claim]:
+    """ Allows bi-directional lookup for claims in a given TokenNetwork. """
+
+    channel_state = get_channelstate_by_token_network_and_partner(
+        chain_state, token_network_address, partner
+    )
+
+    msg = f"Channel with partner {to_checksum_address(partner)} not found in chain_state"
+    assert channel_state, msg
+
+    return channel_state.our_state.claim, channel_state.partner_state.claim
```

Comment:
Not sure if it's worth it to extract that to a separate function if it's called only once.